# SBG Ellipse Messages

Header header

# Time since sensor is powered up (in us)
uint32 time_stamp

# General status bitmask and enums
SbgStatusGeneral status_general

# Communication status bitmask and enums.
SbgStatusCom status_com

# Aiding equipments status bitmask and enums.
SbgStatusAiding status_aiding
